.env.example

```
@@ -0,0 +1,2 @@
+NEXT_PUBLIC_KAKAO_JS_KEY=
+NEXT_PUBLIC_KAKAO_MAP_URL=
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)